packages/ffe-form/less/input-group.less

```diff
@@ -1,7 +1,7 @@
 .ffe-input-group {
     border: 0;
     position: relative;
-    padding: 0 0 calc(1lh + var(--ffe-spacing-xs));
```

Comment:
Ser at det er et tilsvarende problem i AccountSelector. Det settes av `1lh` for å vise kontonummer. Bør vi gjøre noe der også? Synes det uansett ser litt rart ut med tomrom mellom feltet og feilmelding.

Chrome 126:
![Screenshot from 2024-07-10 10-17-17](https://github.com/SpareBank1/designsystem/assets/110812287/e10c8d50-7a24-4287-9fd8-450fd9c3af4f)

Firefox 115:
![Screenshot from 2024-07-10 10-18-00](https://github.com/SpareBank1/designsystem/assets/110812287/de156250-2d37-48f7-976f-f8d780f4c504)

Reply:
Mener jag fikset det slik att ett konto ikke er valgt så legger seg meldningen rakt under. Skall se vad som brukkit det.

Reply:
Det beror på att AccountSelector har emot en ariaInvalid og ikke en `aria-invalid`  som InputGroup legger på children. Detet kan jag ta i en egen sak.

Reply:
Men dette er nok ikke bare og gjøre. Er noen ting man må ta stilling til 

* Hvis vi gjør så att InputGroup også lager ariaInvalid så vill vi få masse elementer som blir slik 
```
<input aria-invalid="true" ariaInvalid="true" />
```
* synes att det gir mening med ariaInvalid i AccountSelector før det er ju ikke ett html element utan en sammansettning av flere. Skall den støte 'aria-invalid' også? Men liker ikke helt att det er props for samme sak egentligen. 

* Skall InputGroup ikke lenger clone children utan tvinga en att bruke funksjon. Dette er ju en massiv BREAKING CHANGE . Jag tror egentligen jag liker denne best førutom att det er veldig stor endring + my mer ansvar blir ju på utviklerna og sette ting riktig, men samtidigt har jag sett mange feil også hvis man glememr seg bort og har flere children så props ikke vidare med nedøver slik det skall. 
```
<InputGroup>
   {props => <input aria-invalid={props.ariaInvalid}  /> }
</InputGroup>
```